dashboard/app/models/script_level.rb

```diff
@@ -45,6 +53,12 @@ def anonymous_must_be_assessment
     end
   end
 
+  def validate_activity_section_lesson
```

Comment:
Is the idea that we want this validation to enforce consistency in the short term, but we'll remove the `stage_id` column / association in the longer term? Or do you think we'll have both indefinitely?

Reply:
I think we can plan to remove the stage_id column in the longer term, once all curriculum is onto the new data model.

there will be some decisions to make about what exactly that will look like. if we decide to bring older lesson plans over, then it may have to wait for that. if we are not bringing older lesson plans over, then we can do the thing we talked about where we just put all the progressions (a.k.a. activity sections) into a single auto-created activity.